udemy_exercises/songs/Main.java

```diff
@@ -0,0 +1,175 @@
+
+import java.util.*;
+
+public class Main {
+
+    private static ArrayList<Album> albums = new ArrayList<Album>();
+
+    public static void main(String[] args) {
```

Comment:
A 'main' function represents the "client" of your package.  Functionality which was expected to be in your Classes, is implemented as client responsibility in our solution.  Due to the exercise description, it is understandable how it led to this approach.  However, you'll come across this often in the "real world" -- requirements which conflict or lead to poor design decisions.  That's when it is important to step back, and establish for yourself a well-organized Object-oriented approach to the design.  In this case, it seems more appropriate for the solution to contain:
 - Song
    > Title
    > Duration
 - Album
    > Artist
    > Title
    > PlayList (immutable)
 - Player
    > PlayList (mutable)

Then, it becomes fairly straightforward to identify in which object specific functionality should reside (i.e. which object should be responsible for this functionality?)